README.md

```diff
@@ -1450,6 +1450,7 @@ See also [Are we game yet?](https://arewegameyet.rs)
   * [fschutt/printpdf](https://github.com/fschutt/printpdf) — PDF writing library
   * [J-F-Liu/lopdf](https://github.com/J-F-Liu/lopdf) — PDF document manipulation
   * [kaj/rust-pdf](https://github.com/kaj/rust-pdf) —
+  * [bastibense/libharu_ng](https://crates.io/crates/libharu_ng) - Easily generate PDFs from your Rust app!
```

Comment:
If it's a `user/repo` link, it should be to the github repo. Crate links have generally been done as just the name of the crate.